BP/common/landed_titles/BP_e_byzantium.txt

```diff
@@ -7950,105 +7858,18 @@ e_byzantium = {
 					color2 = { 255 255 255 }
 				}
 			}
-			c_korinthos = {
-				color = { 185 90 220 }
-				color2 = { 255 255 255 }
-				cultural_names = {
-						roman = cn_corinthus
-						romano_british = cn_corinthus
-						illyro_roman = cn_corinthus
-						thraco_roman = cn_corinthus
-						romano_gothic = cn_corinthus
-						romano_dacian = cn_corinthus
-						romano_hispanic = cn_corinthus
-						romano_vandalic = cn_corinthus
-						romano_aramean = cn_corinthus
-						romano_persian = cn_corinthus
-						romano_frank = cn_corinthus
-						romano_gallic = cn_corinthus
-						african_romance = cn_corinthus
-						dalmatian = cn_corinthus
-						pannonian = cn_corinthus
-						noricroman = cn_corinthus
-						raetoroman = cn_corinthus
-					}
-				#capital = 481 # Corinth
-				
-				b_corinth = {
-					province = 481
-					cultural_names = {
-						roman = cn_corinthus
-						romano_british = cn_corinthus
-						illyro_roman = cn_corinthus
-						thraco_roman = cn_corinthus
-						romano_gothic = cn_corinthus
-						romano_dacian = cn_corinthus
-						romano_hispanic = cn_corinthus
-						romano_vandalic = cn_corinthus
-						romano_aramean = cn_corinthus
-						romano_persian = cn_corinthus
-						romano_frank = cn_corinthus
-						romano_gallic = cn_corinthus
-						african_romance = cn_corinthus
-						dalmatian = cn_corinthus
-						pannonian = cn_corinthus
-						noricroman = cn_corinthus
-						raetoroman = cn_corinthus
-					}
-					color = { 185 90 220 }
-					color2 = { 255 255 255 }
-				}
-				b_nikli = {
-					province = 3732
-					
-					color = { 185 90 220 }
-					color2 = { 255 255 255 }
-				}
-				b_argos = {
-					province = 3798
-					
-					color = { 185 90 220 }
-					color2 = { 255 255 255 }
-				}
-				b_damala = {
-					province = 3799
-					
-					color = { 185 90 220 }
-					color2 = { 255 255 255 }
-				}
-			}
 			c_messenia = {
 				color = { 220 115 220 }
 				color2 = { 255 255 255 }
-				cultural_names = {
-						roman = cn_messene
-						romano_british = cn_messene
-						illyro_roman = cn_messene
-						thraco_roman = cn_messene
-						romano_gothic = cn_messene
-						romano_dacian = cn_messene
-						romano_hispanic = cn_messene
-						romano_vandalic = cn_messene
-						romano_aramean = cn_messene
-						romano_persian = cn_messene
-						romano_frank = cn_messene
-						romano_gallic = cn_messene
-						african_romance = cn_messene
-						dalmatian = cn_messene
-						pannonian = cn_messene
-						noricroman = cn_messene
-						raetoroman = cn_messene
-				}
-				#capital = 3733 # Karytaina
```

Comment:
Same here, seems like a lot of cultural title localisation is being removed?

Reply:
I probably messed that up with a merge conflict after rebasing onto master. Fixed with latest commit.